core/src/main/java/com/sequenceiq/cloudbreak/converter/StackToTemplatePreparationObjectConverter.java

```diff
@@ -308,6 +316,17 @@ private void decorateBuilderWithPlacement(Stack source, Builder builder) {
         builder.withPlacementView(new PlacementView(region, availabilityZone));
     }
 
+    private CustomConfigs getCustomConfigs(Stack source) {
```

Comment:
we should use here a view object and not the real domain. Please create a view like PlacementView and convert here. this modul should be independent from the domain layer